src/app/Instance/InstancePage/InstancePage.tsx

```diff
@@ -126,102 +126,124 @@ const InstancePage = () => {
   ];
 
   return (
-    <>
-      <PageSection variant={PageSectionVariants.light} type="breadcrumb">
-        <Breadcrumb
-          path={[
-            { label: t("instance.smartEventInstances"), linkTo: "/" },
-            { label: instanceName },
-          ]}
-        />
-      </PageSection>
-      <PageSection variant={PageSectionVariants.light}>
-        <Split>
-          <SplitItem isFilled>
-            <TextContent>
-              <Text component="h1">{instanceName}</Text>
-            </TextContent>
-          </SplitItem>
-          <SplitItem>
-            <Dropdown
-              ouiaId="actions-dropdown"
-              onSelect={() => setIsDropdownActionOpen(false)}
-              toggle={
-                <DropdownToggle
-                  ouiaId="actions-dropdown-toggle"
-                  onToggle={(isOpen: boolean) =>
-                    setIsDropdownActionOpen(isOpen)
-                  }
-                  toggleIndicator={CaretDownIcon}
-                >
-                  {t("common.actions")}
-                </DropdownToggle>
-              }
-              isOpen={isDropdownActionOpen}
-              dropdownItems={[
-                <DropdownItem key="details" onClick={onDetailsClick}>
-                  {t("common.details")}
-                </DropdownItem>,
-                <DropdownItem key="delete" onClick={onDeleteClick}>
-                  {t("common.delete")}
-                </DropdownItem>,
-              ]}
+    <Drawer isExpanded={showInstanceDrawer}>
+      <DrawerContent
+        data-ouia-component-id="instance-drawer"
+        panelContent={
+          <InstanceDetails
+            onClosingDetails={() => setShowInstanceDrawer(false)}
+            id="3543edaa-1851-4ad7-96be-ebde7d20d717"
+            endpoint="https://ob-3543edaa-1851-4ad7-96be-ebde7d20d717.apps.openbridge-dev.fdvn.p1.openshiftapps.com/events"
+            name={instanceName}
+            owner="bebianco@redhat.com"
+            published_at="2022-04-12T12:06:22.881959+0000"
+            status="READY"
+            submitted_at="2022-04-12T12:04:43.044590+0000"
```

Comment:
We need almost all of the instance properties for this component. It's reasonable to pass the whole instance as a prop then instead of 6 different ones.